src/Features/DiagnosticsTestUtilities/CodeActionsLegacy/AbstractCodeActionOrUserDiagnosticTest_NoEditor.cs

```diff
@@ -157,7 +157,8 @@ private protected virtual IDocumentServiceProvider GetDocumentServiceProvider()
         protected virtual TestComposition GetComposition()
             => FeaturesTestCompositions.Features
                 .AddExcludedPartTypes(typeof(IDiagnosticUpdateSourceRegistrationService))
-                .AddParts(typeof(MockDiagnosticUpdateSourceRegistrationService));
+                .AddParts(typeof(MockDiagnosticUpdateSourceRegistrationService))
+                .AddAssemblies(typeof(DiagnosticService).Assembly);
```

Comment:
DiagnosticService is in LSP layer, which isn't in the feature layer composition (but is in the editor composition).  i feel like this just makes sense ot be in this composition.